src/components/common/Table/Table.js

```diff
@@ -91,18 +91,18 @@ const TableComponent = ({
   const userObjCreator = () => {
     if (employees) {
       employees.forEach(employee => {
-        const programs = [];
-        employee.programs.forEach(program => {
+        const programsArray = [];
```

Comment:
Nice naming convention--its always good to be very direct about the type of data we are working with.